core/Pos/Core/Constants/Raw.hs

```diff
@@ -200,6 +201,15 @@ isDevelopment = True
 isDevelopment = False
 #endif
 
+#define QUOTED(x) "/**/x/**/"
+
+gitRev :: Text
```

Comment:
How is this going to work? You've moved the function that depends on a environment variable from `node` to `core` which doesn't solve the problem: changing git revision outside will change it and lead to recompilation of core and all dependent modules.